### PR TITLE
Dt node object details

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
### :nut_and_bolt: Description: What code changed, and why?

Three sections have been added to the Chef Infra Server Page:

- Run List
- Manage Tags
- Reset Key

The content, details, and screenshots have been added to the page.

Some old screenshots have also been updated with the new ones (as the UI has been improved).

### :chains: Related Resources

The three PRs which have been taken care of:

- https://github.com/chef/automate/issues/4960
- https://github.com/chef/automate/pull/5086
- https://github.com/chef/automate/pull/5108

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Review by himanshi-chhabra — APPROVED** [review]

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 1]
```diff
@@ -328,7 +331,14 @@ The above information looks like as shown below:
...
 
-The next section of the page is the details section to view or update the node environment. The drop down menu consists of the list of environment created in the chef server. You can select any one environment for the selected node.
+The next section of the page contains two options:
+
+- Details
+- Run List
+
+##### Details
+
+Details section is to view or update the node environment. The drop down menu consists of the list of environment created in the chef server. You can select any one environment for the selected node.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:341]:
```suggestion
The **Details** section lets you view and update the node environment. The drop-down menu consists of the list of environments created in the chef server. You can select any one environment for the selected node.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 2]
```diff
@@ -341,6 +351,48 @@ To select the environment:
 
 {{< figure src="/images/automate/update-node-environment.png" alt="Update Node Environment">}}
 
+##### Run List
+
+You can edit, expand or collapse the run list of a node.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:356]:
```suggestion
You can edit, expand or collapse a node's run list.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 3]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+{{< figure src="/images/automate/run-list-of-a-node.png" alt="Run List of a Node">}}
+
+Chef Infra allows you to edit the run list of a node by following the steps below:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:360]:
```suggestion
Edit a run list by:
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 4]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+1. Select **edit** option. A popup window opens.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:362]:
```suggestion
1. Select **edit**. A popup window opens.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 5]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+2. The left section the window contains the list of environment of the nodes. Select any one environment from the list.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:364]:
```suggestion
2. The left side of the window contains the list of environments. Select an environment from the list.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 6]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+3. The right section of the window will show the current run list of the selected environment.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:366]:
```suggestion
3. The right side of the window will show the run list for the selected environment.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 7]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+4. Select the run list you want for the specific environment and select **Create**.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:368]:
```suggestion
4. Select the run list for a specific environment.
5. Select **Create**.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 8]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+The popup window looks like as shown below:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:370]:
```suggestion
This opens a pop-up window similar to:
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 9]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+{{< figure src="/images/automate/edit-run-list.png" alt="Edit Run List">}}
+
+Chef Infra allows you to edit the run list of a node using a shortcut. Select the option **Edit Run List** from the ellipses icon of the specific node in the node list. Selecting the option will open the same popup window which can be used to edit the run list of a node.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:374]:
```suggestion
You can find the editing window directly by selecting **Edit Run List** from the options list ( {{< fontawesome class="fas fa-ellipsis-h" >}}) of a specific node in the node list. Selecting the option opens a pop-up for editing the run list.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 10]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+The ellipses icon list looks like:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:376]:
```suggestion
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 11]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+{{< figure src="/images/automate/edit-run-list-ellipses-icon.png" alt="Edit Run List Ellipses Icon">}}
+
+#### Manage Tags
+
+Chef Infra allows you to manage tags of the environment. You can add or remove multiple tags by:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:382]:
```suggestion
Chef Infra allows you to manage tags of the environment. You can add or remove multiple tags in two ways:
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 12]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+- Selecting the node and add the tags from the text bar below the dropdown menu of environment.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:384]:
```suggestion
1. Selecting the node and add the tags from the text bar below the dropdown menu of environments.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 13]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:385]:
```suggestion
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 14]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+The popup window looks like as shown below:
+
+{{< figure src="/images/automate/edit-run-list.png" alt="Edit Run List">}}
+
+Chef Infra allows you to edit the run list of a node using a shortcut. Select the option **Edit Run List** from the ellipses icon of the specific node in the node list. Selecting the option will open the same popup window which can be used to edit the run list of a node.
+
+The ellipses icon list looks like:
+
+{{< figure src="/images/automate/edit-run-list-ellipses-icon.png" alt="Edit Run List Ellipses Icon">}}
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:378]: This image isn't very helpful in understanding what the ellipses is. I think it is better to remove it.

File: components/docs-chef-io/content/automate/chef_infra_server.md (hunk 9 above)
Comment (kagarmoe) [reply components/docs-chef-io/content/automate/chef_infra_server.md:374]: I think that should work, but check the build

File: components/docs-chef-io/content/automate/chef_infra_server.md (hunk 14 above)
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:378]:
```suggestion
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 15]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+#### Manage Tags
+
+Chef Infra allows you to manage tags of the environment. You can add or remove multiple tags by:
+
+- Selecting the node and add the tags from the text bar below the dropdown menu of environment.
+
+- OR, by simply selecting the **Manage Tags** option in the ellipses icon of a specific node. Selecting the Manage Tags option from the ellipses icon opens a popup window, where you can add multiple tags and click **Update Tags**. The popup window looks like as shown below:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:386]:
```suggestion
2.  Selecting **Manage Tags** from the options list ( {{< fontawesome class="fas fa-ellipsis-h" >}}) of a specific node. Add and remove tags in tag editor and select **Update Tags** to save your changes.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 16]
```diff
@@ -341,6 +351,48 @@ To select the environment:
...
+
+{{< figure src="/images/automate/manage-tags-from-ellipses-icon.png" alt="Manage Tags">}}
+
+#### Reset a Node Key
+
+You can reset the key of a node by selecting the **Reset Key** option from the ellipses icon of a specific node in the node list. Selecting the option opens a warning popup specifying that the current key will no longer be accepted. Select **Reset Key** once again to confirm. The popup looks like:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:392]: I'm confused about this section. The "node key" isn't a concept in Manage. Should it be client key?

Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:392]:
```suggestion
Reset a node key by selecting **Reset Key** from the options list ( {{< fontawesome class="fas fa-ellipsis-h" >}}) of a specific node in the node list. Selecting **Reset Key** opens a warning that the current key will no longer be accepted. Select **Reset Key** once again to confirm. 
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: components/docs-chef-io/content/automate/chef_infra_server.md (hunk 16 above)
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/chef_infra_server.md]: I have updated the documentation

**Comment by kagarmoe** [comment]
@kalroy This shouldn't have been merged!

**Pull request merged**